docs/extensibility/resources.md

```diff
@@ -62,11 +62,28 @@ The `list` section defines extra columns available in the list. The format is si
 
 - **path** - _[required]_ contains the path to the data used for the column.
 - **widget** - optional widget used to render the field referred to by the `path` property. By default the value is displayed verbatim. For more information about the available widgets, see [Display widgets](display-widgets.md).
+- **formula** - optional formula used to modify data referred to by the `path` property. In `formula` we use the following naming convention: `data.name` instead of just `name`. To learn more about using formulas, see [JSONata](https://docs.jsonata.org/overview.html).
```

Comment:
According to our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-bold-font-for-these-items), parameter names should be **bolded**, not formatted as `code`. 
This applies to the whole documentation.

Reply:
I'll create issue for that